### DISCUSSION TITLE
Can the default service name length in the topology be modified?

### DISCUSSION THREAD

**Discussion by Hu-WF** [apache/skywalking#10772] [Q&A] [answered]
Can the default service name length in the topology be modified? Many service names are not displayed fully, resulting in poor user experience：
![image](https://user-images.githubusercontent.com/30825060/236725795-57817b70-907b-4c08-bf1a-cf6867085dc5.png)

**Comment by wu-sheng** [comment, marked as answer]
What do you mean `can be modified`? This is an open source project, you have all the source codes.

**Comment by Hu-WF** [comment]
Thank you for your quick reply. I would like to know whether it is possible to change the default service name length through environment variables at present. It might be more convenient

**Reply by wu-sheng** [reply]
No, there isn't.
